vic/drivers/image/include/vic_driver_image.h

```diff
@@ -63,6 +70,7 @@ typedef struct {
     size_t n_nx; /**< size of x-index; */
     size_t n_ny; /**< size of y-index */
     location_struct *locations; /**< locations structs for local domain */
+    location_grid_struct *locations_grid; /**< locations structs for full unmasked latlon domain */
```

Comment:
I'm wondering if there is a better way to do this.  Perhaps we could create a `location_struct` for all grid cells, even if they are not being run.  We would just need to add an `active` (or something similar) member to the `location_struct`.  When we do the decomposition, we would only split up active grid cells so nothing would change below the `MASTER_PROC` level.